python/foglamp/plugins/south/common/README.rst

```diff
@@ -1,9 +1,10 @@
-Common South facong plugin code
-===============================
+************************
+Common South Plugin Code
+************************
 
 This directory is the location in which code that is common to more than
-one south facing plugin is located. Code here may not be common to all
-south facing plugins, but it should not be specific to an individual
+one south plugin is located. Code here may not be common to all
```

Comment:
Don't like the phrase "south plugin" personally, it reads as poor English, but I understand why you want it. In a similar way "South service" is poor English to me.